otx/algorithms/common/tasks/training_base.py

```diff
@@ -100,7 +100,8 @@ def __init__(self, task_config, task_environment: TaskEnvironment):
     def _run_task(self, stage_module, mode=None, dataset=None, **kwargs):
         # FIXME: Temporary remedy for CVS-88098
         export = kwargs.get("export", False)
-        self._initialize(export=export)
+        force = kwargs.get("force", False)
+        self._initialize(export=export, force=force)
```

Comment:
Not clear what is being forced here.

Reply:
[Here](https://github.com/cih9088/training_extensions/blob/feature/otx-new-mmcv/mmseg/otx/algorithms/common/tasks/training_base.py#L176-L177).
If `force` is `True`, it initialize the recipe again even if recipe has been initialized before.